app-editors/vim/vim-8.2.0639.recipe

```diff
@@ -10,72 +10,78 @@ configuration files.
 Despite this, Vim can be configured to work in a very simple (Notepad-like) \
 way, called evim or Easy Vim."
 HOMEPAGE="https://www.vim.org/"
-COPYRIGHT="1991-2019 Bram Moleenar et al."
+COPYRIGHT="1991-2020 Bram Moleenar et al."
 LICENSE="Vim"
-REVISION="4"
+REVISION="1"
 SOURCE_URI="https://github.com/vim/vim/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="3bfb6b0de36e96862933e49df93412a46a0914356ec16dd60a5ccbd5cdc46d25"
+CHECKSUM_SHA256="5ec230131a9dff131fa199f47fd391341625d0bad056aa3eed970b70628886ae"
 SOURCE_FILENAME="vim-$portVersion.tar.gz"
-PATCHES="vim-$portVersion.patchset"
 
-ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86_64"
```

Comment:
Without patching it won't work for x86_gcc2 so:
```diff
- ARCHITECTURES="x86_gcc2 x86_64"
+ ARCHITECTURES="!x86_gcc2 x86_64"
```

Reply:
Fixed!